gimbal-adapter/src/main/java/com/urbanairship/gimbal/GimbalAdapter.java

```diff
@@ -207,27 +195,16 @@ public void removeListner(Listener listener) {
      */
     @RequiresPermission(ACCESS_FINE_LOCATION)
     public boolean start(@NonNull String gimbalApiKey) {
-        if (isStarted) {
-            return true;
-        }
-
-        Gimbal.setApiKey((Application) context.getApplicationContext(), gimbalApiKey);
-
-        if (!isPermissionGranted()) {
```

Comment:
Do we need to check if permission is granted? What does `Gimbal.start()` do when permissions are not granted? Looks like `PlaceManager.getInstance().startMonitoring();` in the docs say we need to check permissions before starting it.